### PR TITLE
Overview Page

### PR DESCRIPTION
**Pull request by psankhe28**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -26,27 +46,31 @@ function App() {
...
         <>
           <Loader loading={showLoader} />
           <MDBRow>
-           {user && <MDBCol size={2} className="p-0">
-              <SidebarComponent />
-            </MDBCol>}
-            <MDBCol size={user ? 10 : 12}>
+            {user && (
+              <MDBCol size={collapsed ? 1 : 2} className="p-0">
+                <SidebarComponent
```
Comment (geeky-abhishek) [inline src/App.js]: why are be handling collapsed from the App.js when we can handle this directly in the sidebar component

File: src/pages/monitoring/overview/index.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from "react";
+import { MDBCard, MDBCardBody } from "mdb-react-ui-kit";
+import "./style.css";
+import { useStore } from "../../../store";
+
+type Theme = "light" | "dark";
```
Comment (geeky-abhishek) [inline src/pages/monitoring/overview/index.tsx]: along with this type, can you plz move all the types to the type folder?

File: src/pages/monitoring/overview/index.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+export const Overview = () => {
+  const store: any = useStore();
+  const [theme, setTheme] = React.useState<Theme>(store?.theme);
+
```
Comment (geeky-abhishek) [inline src/pages/monitoring/overview/index.tsx]: don't add theme management in separate components but instead manage it for all components from the sidebar only

File: src/pages/monitoring/overview/index.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  useEffect(() => {
+    setTheme(store?.theme);
+  }, [store?.theme]);
+
+  return (
+    <div className="container my-3 text-center align-items-center">
```
Comment (geeky-abhishek) [inline src/pages/monitoring/overview/index.tsx]: there are proper MDB component for Container,Row,Col you can use theme directly

File: src/App.js (hunk 1 above)
Comment (psankhe28) [reply src/App.js]: When the sidebar collapses, we need to use the space which was occupied by the sidebar for the main page body. So, having it in App.js helps to do that.

**Review by chinmoy12c — APPROVED** [review]
Please remove the package-lock.json file changes from the PR.

File: src/pages/monitoring/overview/index.tsx (hunk 2 above)
Comment (psankhe28) [reply src/pages/monitoring/overview/index.tsx]: Can you please give more details about this?

**Pull request closed without merging**